### PR TITLE
UI remove query-string package and unused commit arg

### PR DESCRIPTION
**Pull request by nopcoder**
Close https://github.com/treeverse/lakeFS/issues/7128

### CODE REVIEW

File: webui/src/lib/api/index.js [hunk 1]
```diff
@@ -26,6 +24,11 @@ class LocalCache {
 
 const cache = new LocalCache();
 
+export const qs = (queryParts) => {
+    const parts = Object.keys(queryParts).map(key => [key, queryParts[key]]);
```
Comment (arielshaqed) [inline webui/src/lib/api/index.js:28]: Can we use [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) here, and trust our transpiler to do the right thing for all browsers?
```suggestion
    const parts = Object.entries(queryParts);
```

**Review by arielshaqed — APPROVED** [review]
Thanks!  Would rather have reviewed this as two PRs.

File: webui/src/lib/api/index.js (hunk 1 above)
Comment (nopcoder) [reply webui/src/lib/api/index.js:28]: Don't think we have one. But we already use `Object.entries` in our code and the global adoption for this API is on 97.26%.
So I'll take it, thanks!

**Comment by nopcoder** [comment]
> Thanks! Would rather have reviewed this as two PRs.

Assume you are talking about the code vs remove the package from package*.json.
Thought because it is a single package this PR will hold.

**Pull request merged**